pom.xml

```diff
@@ -24,12 +24,18 @@
     </scm>
     <properties>
         <datawave.webservice.namespace>http://webservice.datawave/v1</datawave.webservice.namespace>
+        <version.annotations>2.0.3</version.annotations>
         <version.datawave>7.3.0</version.datawave>
-        <version.datawave.hazelcast-client>4.0.2</version.datawave.hazelcast-client>
+        <version.datawave.hazelcast>4.0.2</version.datawave.hazelcast>
```

Comment:
We might not want to change this from `<version.datawave.hazelcast-client>` to `<version.datawave.hazelcast>`. Seems like we are still using `hazelcast-client` in the pom.

In the latest standardization of property names effort, `hazelcast-common` was actually the one that got renamed to just hazelcast. `hazelcast-client` remained unchanged. So, for the sake of consistency we probably don't want to change this. Might mean we need to add another property here.